src/Microsoft.Health.Dicom.Functions.Abstractions/Update/UpdateInput.cs

```diff
@@ -4,12 +4,13 @@
 // -------------------------------------------------------------------------------------------------
 
 using System.Collections.Generic;
+using Microsoft.Health.Dicom.Core.Features.Partitioning;
 
 namespace Microsoft.Health.Dicom.Functions.Update;
 
 public class UpdateInput
 {
-    public int PartitionKey { get; set; }
+    public Partition Partition { get; set; }
```

Comment:
We can do this as long as there aren't active update jobs. Otherwise the change must be additive until all active jobs use the name schema

Reply:
Zeiss can test update at any moment